android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -1084,11 +1072,6 @@ public void onBalance(int errorCode) {
                 TextView usdBalanceText = mPopupView.findViewById(R.id.usd_balance_text);
                 usdBalanceText.setText(usdText);
             }
-        } else if (errorCode == BraveRewardsNativeWorker.LEDGER_ERROR) { // No Internet connection
-            String args[] = {};
-            Log.e(TAG, "Failed to fetch rewards parameters from server");
-            showNotification(
-                    REWARDS_NOTIFICATION_NO_INTERNET_ID, REWARDS_NOTIFICATION_NO_INTERNET, 0, args);
```

Comment:
we may need to show different notification for LEDGER_ERROR. CC @Miyayes

Reply:
@deeppandya I don't think we show anything on desktop, so we don't need to show anything here?

Reply:
Yeah, it doesn't seem necessary to show anything here (desktop doesn't, from what I can see).

Reply:
@deeppandya Okay, then good to remove without replacement. Thanks.